CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
 # Unreleased Changes
+
+## Unreleased: v4.0.0
+
+#### Bug fixes:
+- Fixed issue #132 get_default_settings in API200 is currently an instance method, but it should be a class method
```

Comment:
I think we need to make it clear that it is a breaking change. In the CHANGELOG notes for v2.0.0 we added in a section for breaking changes, and I think it would be wise to do that here too (Rename the section from `Bug fixes` to `Breaking changes`).

Also, just to make it clear that a v4.0.0 version doesn't actually exist, I suggest we say "Suggested" or "Expected" release v4.0.0. The way it is currently makes it seem like there is a beta version or something of 4.0.0 out but just hasn't been released yet.